insta-ui/src/server/prompts.ts

```diff
@@ -43,5 +44,59 @@ export function generateMessages(textInput: string, imageInput?: string, previou
         });
     }
 
-    return messages;
-}
\ No newline at end of file
+    const topicMessages: Message[] = []
+    if (topicName) {
+        topicMessages.push({
+            role: 'system',
+            content: `You are a request title creator. You have to help the user summarize a request in no more than 6 words.`
+        });
+
+        const userContent: (TextContent | ImageContent)[] = [
+            { type: 'text', text: `Create a request title based on this description: ${textInput}` }
+        ];
+    
+        if (imageInput) {
+            userContent.push({
+                type: 'image_url',
+                image_url: {
+                    url: imageInput
+                }
+            } as ImageContent);
+        }
+        
+        topicMessages.push({
+            role: 'user',
+            content: userContent
+        });
+    
+    }
+
+    return {messages, topicMessages};
+}
+
+// export function generateReviseMessages(code: string): Message[] {
+//     const messages: Message[] = [
+//         {
+//             role: 'system',
+//             content: `You are a React component code debugger. Keep the original structure and style of the code as much as possible, detect any errors, and change it into valid, compilable React code.  
+//       Follow these requirements:
+//       - Use only React and standard packages
+//       - Component must be compilable
+//       - Include 'import React' at the start
+//       - Use 'export default' for the component
+//       - Ensure proper JSX syntax`
+//         }
+//     ];
+
+//     // Add user message with text and optionally image
+//     const userContent: (TextContent | ImageContent)[] = [
+//         { type: 'text', text: `Revise the provided code: ${code}` }
+//     ];
+
+//     messages.push({
+//         role: 'user',
+//         content: userContent
+//     });
+
+//     return messages;
+// }
```

Comment:
You can safely delete the comments to keep the codebase clean :) We can always retrieve it from git history

Reply:
Sure, I'll delete them